docs/data/material/customization/density/DensityTool.js

```diff
@@ -86,7 +101,11 @@ export default function DensityTool() {
               'aria-labelledby': 'input-slider',
             }}
           />
-          <IconButton aria-label={t('decreaseSpacing')} onClick={increaseSpacing}>
+          <IconButton
+            aria-label={t('decreaseSpacing')}
```

Comment:
Not from your changes.
```suggestion
            aria-label={t('increaseSpacing')}
```

Reply:
Done.